### PR TITLE
fix: Display the updated qty and total for refunded items

### PR DESCRIPTION
**Pull request by danielbreves**
This seems more consistent with how Woocommerce displays the order for admins:
Screen Shot 2020-09-09 at 6 45 08 pm

### CODE REVIEW

**Comment by alamgircsebd** [comment]
@kapilpaul vai please take a look on this pull request

**Comment by kapilpaul** [comment]
@alamgircsebd  will check this one.

**Comment by Tangol** [comment]
@kapilpaul Any news on this?
Thanks!

File: templates/orders/order-item-html.php [hunk 1]
```diff
@@ -79,7 +79,19 @@
...
+                $refunded_qty = $order->get_qty_refunded_for_item( $item_id );
+
+                if ( $refunded_qty ) {
+                    echo '<del>' . esc_html( $item_qty ) . '</del> ';
+                    $item_qty = $item_qty + $refunded_qty; // $refunded_qty is already negative
+                }
+
+                echo esc_html( $item_qty );
+            }
+        ?>
```
Comment (alamgircsebd) [inline templates/orders/order-item-html.php:94]: Replace the codes with this:


```
        $qty                  = $item->get_quantity();
        $refunded_qty = $order->get_qty_refunded_for_item( $item_id );

        if ( $refunded_qty ) {
            $qty_display = '<del>' . esc_html( $qty ) . '</del> <ins>' . esc_html( $qty - ( $refunded_qty * -1 ) ) . '</ins>';
        } else {
            $qty_display = esc_html( $qty );
        }

        echo $qty_display;
```

**Comment by shashwatahalder01** [comment]
Since this feature is already implemented on dokan pro and refund is a pro feature, we are not accepting this PR. Currently, we are not planning to move this feature from dokan pro to dokan lite.

**Pull request closed without merging**